resources/js/Components/SearchBars/UserSearch.vue

```diff
@@ -14,11 +14,23 @@ export default {
             users: []
         }
     },
+    props: {
+        onlyManager: {
+            type: Boolean,
+            default: false
+        }
+    },
     emits: ['user-selected'],
     methods: {
         selectUser(user) {
             this.$emit('user-selected', user);
             this.user_search_query = '';
+        },
+        checkIfOnlyManager(user) {
+            if (this.onlyManager) {
+                return user?.project_manager_permission;
```

Comment:
MUST: Das ? muss entfernt werden, der User ist in jedem Fall vorhanden. So wird suggeriert, dass der user leergelassen werden kann.